server/profileGet.php

```diff
@@ -0,0 +1,31 @@
+<?php
+$root = realpath($_SERVER["DOCUMENT_ROOT"]);
+
+require_once "$root/server/user.class.php";
+$USER = new User();
+if (!$USER->hasPermission('can_use_admin_tools')) {
+  header("Location: /unauthorized");
+  die();
+}
+require_once 'config.php';
+getProfile($USER->getUserId());
+function getProfile($data) {
+  GLOBAL $DB_CONN:
```

Comment:
This line needs to end in a semicolon, not a colon. As an aside, errors are the worst in PHP. Any sort of error, syntax/compile error, real error, etc. They all just make the server respond 500 with like next-to-nothing to help with debugging. But I find that the majority of the errors I make are syntax errors, so if you ever see the 500 in the JS console, try making sure your syntax is correct everywhere first.